pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -392,6 +397,65 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	return true, nil
 }
 
+func checkLibvirtCrcBridgePermissions() (bool, error) {
+	logging.Debug("Checking if 'crc' bridge has appropriate permissions setup")
+	configPath := ""
+	for _, path := range qemuBridgeConfigPaths {
+		logging.Debug(fmt.Sprintf("Trying %s..", path))
+		_, err := os.Stat(path)
+		if err != nil {
+			logging.Debug(fmt.Sprintf("Failed to open %s: %s", path, err))
+		} else {
+			configPath = path
+
+			break
+		}
+	}
+	if configPath == "" {
+		return false, fmt.Errorf("Failed to find Qemu bridge configuration file")
+	}
+
+	config, err := ioutil.ReadFile(filepath.Clean(configPath))
+	if err != nil {
+		return false, fmt.Errorf("Failed to read %s: %v", configPath, err)
+	}
+	if match, _ := regexp.MatchString(qemuBridgeConfig, string(config)); !match {
+		return false, fmt.Errorf("`crc` network not allowed unprivileged access")
+	}
+	logging.Debug("'crc' bridge has appropriate permissions")
+	return true, nil
+}
+
+func fixLibvirtCrcBridgePermissions() (bool, error) {
+	logging.Debug("Fixing permissions for 'crc'")
+	configPath := ""
+	for _, path := range qemuBridgeConfigPaths {
+		logging.Debug(fmt.Sprintf("Trying %s..", path))
+		_, err := os.Stat(path)
+		if err != nil {
+			logging.Debug(fmt.Sprintf("Failed to open %s: %s", path, err))
+		} else {
+			configPath = path
+
+			break
+		}
+	}
+	if configPath == "" {
+		return false, fmt.Errorf("Failed to find Qemu bridge configuration file")
+	}
+
+	err := crcos.AppendToFileAsRoot(
+		"Allow 'crc' network to be used from session libvirt",
+		fmt.Sprintf("%s\n", qemuBridgeConfig),
+		configPath,
+	)
+	if err != nil {
+		return false, fmt.Errorf("Failed to write to %s: %v", configPath, err)
+	}
+	logging.Debug("'crc' bridge now has appropriate permissions")
```

Comment:
Something like "The 'crc' bridge can now be used by qemu-bridge-helper/session libvirt" would be more accurate. We are not really changing 'permissions' on anything.